.circleci/common.yml

```diff
@@ -180,6 +178,60 @@ commands:
             rc="${rc}}"
             echo "$rc" > ~/.terraformrc
 
+  setup_github_bot:
+    steps:
+      - run:
+          name: Get Github Bot Token
+          command: |
+            app_id=$GITHUB_APP_ID
```

Comment:
Sadly I couldn't find an orb to do this for us. But this script will take the Github Bot secrets and convert it to a regular access token like we already use, valid for 10 min.

i wish CircleCI let us ask for a github access token like Github Actions do, then this wouldnt be necessary 🤷‍♂️

Reply:
Cool, we can make this an orb of our own later?